packages/dataviews/src/types.ts

```diff
@@ -216,7 +216,7 @@ interface ViewBase {
 	/**
 	 * The hidden fields.
 	 */
-	hiddenFields: string[];
+	hiddenFields?: string[];
```

Comment:
I've been reading our types lately and this one actually stood out to me :P 

I think it makes more sense to have "visibleFields" rather than "hiddenFields" as we move forward. I'm pretty sure most entities will have a lot more fields that what we currently have and an allow list would be better.

Omitting it could have the same effect as today when you omit hiddenFields.

Not a concern for this PR though, just a thought that I had.

Reply:
Yeah, that's a good point: post types have a lot of metadata and if we aim to share the field definition between a view and a form (e.g.: details panel), a lot of the fields would be hidden in the view.